roles/networking/tasks/main.yml

```diff
@@ -7,5 +7,8 @@
 - include: redhat.yml
   when: ansible_os_family == "RedHat"
 
-- include: ubuntu.yml
-  when: ansible_distribution == "Ubuntu"
+- include: ubuntu_16.yml
+  when: ansible_distribution == "Ubuntu" and ansible_distribution_release != 'bionic'
```

Comment:
Can we test for xenial rather than !bionic?

Reply:
I'm testing for not bionic because we support 14.0.4 and 16.04 and this works for that condition.

Reply:
We don't support 14.04. We should build for 16.04 and 18.04 only at this point.

Reply:
makes sense, will fix accordingly.